core/include/logger/writer_full.h

```diff
@@ -50,6 +50,13 @@ class WriterFull : public QObject
      */
     void flush(const QString f_entry);
 
+    /**
+     * @brief Writes log entry into area seperated logfiles.
+     * @param Preformatted QString which will be written into the logfile
+     * @param Area name of the target logfile.
+     */
+    void flushArea(const QString f_entry, const QString f_areaName);
```

Comment:
```suggestion
    void flush(const QString f_entry, const QString f_areaName);
```
overload flush() instead of making a different function for it